app/javascript/mastodon/components/status_action_bar.jsx

```diff
@@ -9,7 +9,7 @@ import ImmutablePropTypes from 'react-immutable-proptypes';
 import ImmutablePureComponent from 'react-immutable-pure-component';
 import { connect } from 'react-redux';
 
-import BookmarkIcon from '@/material-icons/400-24px/bookmark-fill.svg';
+import BookmarkIcon from '@/material-icons/400-24px/bookmark-fill.svg?react';
```

Comment:
Extremely minor, but this could be pulled out to another PR.

Reply:
i don't think it's a problem with Webpacker, but it caused a fatal with Vite